common/src/main/java/tachyon/Constants.java

```diff
@@ -157,6 +158,15 @@
   public static final String MASTER_RESOURCE_CPU = "tachyon.master.resource.cpu";
   public static final String MASTER_RESOURCE_MEM = "tachyon.master.resource.mem";
 
+  public static final String MASTER_CHECKPOINT_STRATEGY_CLASS =
```

Comment:
by our convention, this should be called
`MASTER_LINEAGE_CHECKPOINT_STRATEGY_CLASS`,
also for the following three